build failure on Gentoo: 'strerror' was not declared in this scope

`man strerror` says it is defined in `<string.h>`

```
[ 11%] Building CXX object library/lib/CMakeFiles/Pothos.dir/Framework/ThreadConfigUnix.cpp.o
/home/gentoo/src/radio/pothos/library/lib/Framework/ThreadConfigUnix.cpp: In static member function 'static std::string ThreadEnvironment::setPriority(double)':
/home/gentoo/src/radio/pothos/library/lib/Framework/ThreadConfigUnix.cpp:25:43: error: 'strerror' was not declared in this scope
     if (maxPrio < 0) return strerror(errno);
                                           ^
/home/gentoo/src/radio/pothos/library/lib/Framework/ThreadConfigUnix.cpp:27:43: error: 'strerror' was not declared in this scope
     if (minPrio < 0) return strerror(errno);
                                           ^
/home/gentoo/src/radio/pothos/library/lib/Framework/ThreadConfigUnix.cpp:31:5: error: 'memset' is not a member of 'std'
     std::memset(&param, 0, sizeof(param));
     ^
/home/gentoo/src/radio/pothos/library/lib/Framework/ThreadConfigUnix.cpp:36:74: error: 'strerror' was not declared in this scope
     if (sched_setscheduler(0, policy, &param) != 0) return strerror(errno);
                                                                          ^
/home/gentoo/src/radio/pothos/library/lib/Framework/ThreadConfigUnix.cpp: In static member function 'static std::string ThreadEnvironment::setCPUAffinity(const std::vector<long unsigned int>&)':
/home/gentoo/src/radio/pothos/library/lib/Framework/ThreadConfigUnix.cpp:59:76: error: 'strerror' was not declared in this scope
     if (sched_setaffinity(0, size, cpusetp) != 0) errorMsg = strerror(errno);
                                                                            ^
/home/gentoo/src/radio/pothos/library/lib/Framework/ThreadConfigUnix.cpp: At global scope:
/home/gentoo/src/radio/pothos/library/lib/Framework/ThreadConfigUnix.cpp:67:13: warning: unused parameter 'affinity' [-Wunused-parameter]
 std::string ThreadEnvironment::setNodeAffinity(const std::vector<size_t> &affinity)
             ^
library/lib/CMakeFiles/Pothos.dir/build.make:77: recipe for target 'library/lib/CMakeFiles/Pothos.dir/Framework/ThreadConfigUnix.cpp.o' failed
make[2]: *** [library/lib/CMakeFiles/Pothos.dir/Framework/ThreadConfigUnix.cpp.o] Error 1
CMakeFiles/Makefile2:232: recipe for target 'library/lib/CMakeFiles/Pothos.dir/all' failed
make[1]: *** [library/lib/CMakeFiles/Pothos.dir/all] Error 2
Makefile:126: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/library/lib/Framework/ThreadConfigUnix.cpp
+++ b/library/lib/Framework/ThreadConfigUnix.cpp
@@ -9,6 +9,7 @@
 #include <numa.h>
 #endif
 #include <cerrno> //errno
+#include <cstring> //strerror
 
 #ifdef __APPLE__
 #include <thread>

